DIPs/archive/DIP75.md

```diff
@@ -0,0 +1,75 @@
+# Release Process
+
+| Section        | Value                                                         |
+|----------------|------------------------------------|
+| DIP:           | 75                                 |
+| Status:        | Implemented                        |
+| Author:        | David Soria Parra and Martin Nowak |
+
+## Abstract
+
+The current release process of DMD, Druntime and Phobos (from now on referred to as "release package"), is done on a "done when it's done" basis. This DIP proposes a time-based release schedule. This allows:
+
+-   reduced times to get bugfixes out
+-   a clear way of managing backwards compatibility
+-   a more predictable release schedule for packagers
+-   cleaner responsibilities
+-   assurance to developers regarding timeliness of release of their contributions
+-   synchronization with releases processes of major distributions
+
+In addition the DIP proposes the idea of **Release Managers**, who are responsible for a timely release.
+
+## Release Process
+
+We propose to introduce **bugfix versions** and **feature versions**. A **feature version** may contain new features, language changes, and deprecations. For every release version the minor part of the version is incremented: 2.066, 2.067, ... A **bugfix version** is based on a feature version and contains only bugfixes and neither new features, nor deprecations of other features or library changes. Bugfix versions are for example: 2.066.1, 2.066.2, etc. There is only one supported version at a time, but release managers are free to release additional bugfix versions when needed.
+
+### Schedule
+
+A bugfix version is released *every month*. A feature version is released every 2 months. Releases should happen on the first Saturday of the month. Two weeks before a feature release *stable* is branched off *master*. No new features are integrated into *stable*.
```

Comment:
> Ideally, we'd focus our team effort on keeping the schedule. I hardly had time the past few month and nobody else was working on those blockers.

Well my knowledge is a bit too limited too work on DMD atm (and I have my official GSoC project), but you should definitely try to make some noise and get attention to this problem.
Here are some additional ideas, they might not work though:

1) "Bug of the week" (Blocking bugs -> posted for more awareness)
2) Attach some money to the bugs (I know this has been tried before and didn't work so good, but it might still be worth continuing to try. I believe very few people know about open bounties (https://www.bountysource.com/teams/d/issues)